Makefile

```diff
@@ -3,25 +3,24 @@ LEKTOR_SERVER_FLAGS=-h 127.0.0.1
 all: build
 
 sass:
-	sass -t compressed ./assets/sass/main.scss ./assets/css/main.min.css
-	sass -t compressed ./assets/sass/ie9.scss ./assets/css/ie9.min.css
-	rm ./assets/css/main.min.css.map
-	rm ./assets/css/ie9.min.css.map
+	scss ./assets/sass/main.scss ./assets/css/main.min.css
+	scss ./assets/sass/ie9.scss ./assets/css/ie9.min.css
```

Comment:
Don't look like this would generate a minified version of css.

Reply:
Thats already known but thats definetly the smallest problem.